REQUIRE

```diff
@@ -1,5 +1,6 @@
-julia 0.5
+julia 0.6
 JSON 0.6.0
 MathProgBase 0.5.4
 JuMP 0.17 0.19-
-Compat 0.17
+Compat 0.26.0
```

Comment:
Did you check this is the min version of compat?  I often get pinged on this when tagging new versions.

Reply:
Yeah, [0.26.0](https://github.com/JuliaLang/Compat.jl/releases/tag/v0.26.0) was when `@__MODULE__` was added to Compat which is the only reason I needed to bump the version number.